Scripts/script-CompleteManualTask.yml

```diff
@@ -0,0 +1,29 @@
+commonfields:
+  id: CompleteManualTask
+  version: -1
+name: CompleteManualTask
```

Comment:
This looks like script that is relevant only for tests, then move it to be under `TestPlaybooks ` folder

Reply:
Done